src/ar_http_iface_tests.erl

```diff
@@ -736,6 +736,28 @@ get_txs_by_send_recv_test_() ->
 			))
 	end}.
 
+get_tx_status_test() ->
+	ar_storage:clear(),
+	SearchServer = app_search:start(),
+	Peers = ar_network:start(10, 10),
+	ar_node:add_peers(hd(Peers), SearchServer),
+	TX = (ar_tx:new())#tx {tags = [{<<"TestName">>, <<"TestVal">>}]},
+	ar_node:add_tx(hd(Peers), TX),
+	receive after 250 -> ok end,
+	ar_node:mine(hd(Peers)),
+	receive after 1000 -> ok end,
+	{ok, {_, _, Body, _, _}} =
+		ar_httpc:request(
+			<<"GET">>,
+			{127, 0, 0, 1, 1984},
+			"/tx/" ++ binary_to_list(ar_util:encode(TX#tx.id)) ++ "/status"
+		),
+	{Res} = ar_serialize:dejsonify(Body),
+	HashList = ar_node:get_hash_list(hd(Peers)),
+	?assertEqual(true, lists:member({<<"block_height">>, length(HashList) - 1}, Res)),
+	?assertEqual(true, lists:member({<<"block_indep_hash">>, ar_util:encode(hd(HashList))}, Res)),
+	?assertEqual(true, lists:member({<<"number_of_confirmations">>, 0}, Res)).
```

Comment:
If we let it mine a few more blocks, we could also verify that this value increased correctly. Either we mine before these asserts, or we add `ar_node:mine/1` calls after these asserts and do the request again.